lib/AST/Decl.cpp

```diff
@@ -5042,3 +5042,20 @@ getObjCNameForSwiftDecl(const ValueDecl *VD, DeclName PreferredName){
     return {Ctx.getIdentifier(getNameForObjC(VD)), ObjCSelector()};
   }
 }
+
+bool swift::objc_translation::
+isVisibleToObjC(const ValueDecl *VD, Accessibility minRequiredAccess,
```

Comment:
It's generally preferable for headers to have an associated implementation file, so `SwiftNameTranslation.h` can have `SwiftNameTranslation.cpp` for its code, it doesn't have to go to `Decl.cpp`